libraries/stdlib/common/src/kotlin/KotlinH.kt

```diff
@@ -9,11 +9,29 @@ import kotlin.annotation.AnnotationTarget.FIELD
 import kotlin.annotation.AnnotationTarget.PROPERTY
 
 
-
+/**
+ * Provides a [comparison] function for imposing a total ordering between instances of a class.
```

Comment:
I'll change this to the following upon merge:
```suggestion
 * Provides a comparison function for imposing a total ordering between instances of the type [T].
```